Migrations/Mysql/Version20250721210441.php

```
@@ -0,0 +1,29 @@
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250721210441 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)